marklogic-data-hub-central/ui/src/components/entities/matching/matching-step-detail/matching-step-detail.tsx

```diff
@@ -753,9 +753,10 @@ const MatchingStepDetail: React.FC = () => {
           </div>
           <div><span><b>Enable Threshold Scale </b></span><Switch aria-label="threshold-scale-switch" onChange={(e) => toggleDisplayThresholdTimeline(e)} defaultChecked={false} ></Switch>
             <span>
-              <Tooltip title={MatchingStepTooltips.thresholdScale} placement={"right"}>
-                <Icon type="question-circle" className={styles.scaleTooltip} theme="filled" data-testid={"info-tooltip-threshold"} />
-              </Tooltip><br />
+              <HCTooltip text={MatchingStepTooltips.thresholdScale} id="threshold-scale-tooltip" placement="right">
```

Comment:
@federicoRufrancosMarklogic  I don't see any reference to this identifier `threshold-scale-tooltip` in the code (tests, css or js).
I'm wondering why do we have `id` as requiered in `hc-tooltip`?

Reply:
@romanlescano yes it's required on hctooltip because then the react bootstrap tooltip require an id.
https://react-bootstrap.github.io/components/overlays/#tooltip-props